Functions.Templates/Templates/HttpTrigger-CSharp/HttpTriggerCSharp.cs

```diff
@@ -37,9 +37,11 @@ public static class HttpTriggerCSharp
             dynamic data = JsonConvert.DeserializeObject(requestBody);
             name = name ?? data?.name;
 
-            return name != null
-                ? (ActionResult)new OkObjectResult($"Hello, {name}")
-                : new BadRequestObjectResult("Please pass a name on the query string or in the request body");
+            string responseMessage = name != null

```

Comment:
Just a note that `name != null` will miss the empty/whitespace string case, which the F# trigger does account for. Might be worth doing the same here?

Reply:
Fixed